src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,15 @@
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
box-sizing and font-family are normalize more 😅

Reply:
> 

🤯  😅